js/index.js

```diff
@@ -1,7 +1,77 @@
-// Iteration 1: Names and Input
-// 
+console.log("I am ready!");
 
-// Iteration 2: Conditionals
+// Iteration1
+let hacker1 = "James";
+console.log(`The driver's name is ${hacker1}`);
 
+let hacker2 = "Peter";
+console.log(`The navigator's name is ${hacker2}`);
 
-// Iteration 3: Loops
\ No newline at end of file
+// Iteration2
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters.`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters.`
+  );
+} else if (hacker1.length === hacker2.length) {
+  console.log(
+    `Wow, you both have equally long names, ${hacker2.length} characters!`
+  );
+}
```

Comment:
Just so you know: You could also write if, else if, else like this:
...
else  {
  console.log(
    `Wow, you both have equally long names, ${hacker2.length} characters!`
  );